snuba/query/query.py

```diff
@@ -79,6 +182,9 @@ def __extend_sequence(self,
     def get_selected_columns(self) -> Optional[Sequence[Any]]:
         return self.__body.get("selected_columns")
 
+    def get_selected_columns_exp(self) -> Sequence[Expression]:
```

Comment:
This suffix is kind of misleading — this isn't an expression, it's a sequence of expressions. `_exprs` (or even `_expressions`) is probably more appropriate here (and some of the methods below.)